Source/WebCore/css/parser/CSSTokenizerInputStream.cpp

```diff
@@ -42,18 +44,35 @@ CSSTokenizerInputStream::CSSTokenizerInputStream(const String& input)
 
 void CSSTokenizerInputStream::advanceUntilNonWhitespace()
 {
-    // Using ASCII whitespace here rather than CSS space since we don't do preprocessing
```

Comment:
What’s the rationale for removing this comment?

Reply:
Was erroneously removed, I've added it back.